src/pretix/base/migrations/0251_logentry_organizer.py

```diff
@@ -0,0 +1,56 @@
+# Generated by Django 4.2.4 on 2023-11-20 12:38
+
+from django.db import migrations, models
+import django.db.models.deletion
+from django.db.models import F, Subquery, OuterRef
+
+
+def backfill_organizer(apps, schema_editor):
+    LogEntry = apps.get_model("pretixbase", "LogEntry")
+    Event = apps.get_model("pretixbase", "Event")
+    ContentType = apps.get_model("contenttypes", "ContentType")
+
+    LogEntry.objects.filter(
+        organizer_link__isnull=True, event__isnull=False
+    ).update(organizer_link_id=Subquery(
+            Event.objects.filter(pk=OuterRef('event_id')).values('organizer_id'),
+        )
+    )
+    for ct in ContentType.objects.all():
+        try:
+            model = apps.get_model(ct.app_label, ct.model)
+        except LookupError:
+            continue
+        if "organizer" in model._meta.fields:
+            LogEntry.objects.filter(
+                organizer_link__isnull=True, event__isnull=False, content_type=ct,
```

Comment:
Do we need the `event__isnull=False` here? We already have the organizer in the model so we probably can leave that out? Or should we even do ``event__isnull=True`` as we already updated all LogEntries that have an event attached in line 14-16.

Also, having ``event__isnull=True`` would update LogEntries, that are related to a model with an organizer, but not an event. Not sure, we have logs for this though …

Reply:
I think this should have been event__isnull=True, yes

Reply:
> Also, having event__isnull=True would update LogEntries, that are related to a model with an organizer, but not an event. Not sure, we have logs for this though …

Yes, like a Customer account?